content/it/docs/reference/glossary/kubeadm.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Kubeadm
+id: kubeadm
+date: 2018-04-12
+full_link: /docs/admin/kubeadm/
+short_description: >
+  Un tool per installare velocemente Kubernetes e avviare un cluster sicuro.
+
+aka: 
+tags:
+- tool
+- operation
+---
+ Un tool per installare velocemente Kubernetes e avviare un cluster sicuro.
+
+<!--more--> 
+
+Tu puoi usare kubeadm per installare sia il control plane che il {{< glossary_tooltip text="worker node" term_id="node" >}}.
```

Comment:
I would remove the "Tu", since feels redundant.
The "control plane" has been considered female in Italian in other translation in previous pages and in other pages of this PR as well.